.ci.yaml

```diff
@@ -1143,6 +1147,10 @@ targets:
         {
           "CHANNEL": "master"
         }
+      dependencies: >
+        [
+          {"dependency": "swift_format", "version": "build_id:8797338979890974865"}
```

Comment:
The arm64 and x64 `build_id` numbers are different, but it's pointing to the same version.
https://github.com/flutter/packages/pull/5928#discussion_r1457919953